client/webserver/site/package.json

```diff
@@ -1,48 +1,45 @@
 {
   "name": "dexclient",
-  "version": "1.0.0",
-  "description": "",
+  "version": "1.1.0",
+  "description": "dexc browser frontend",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "webpack --watch --config webpack/dev.js",
```

Comment:
`npm run watch` is broken for me.

Reply:
Whoops.  I swear I tested `run watch`, must have been before I added the `path` change as per the webpack 5 migration guide.

Just as well because apparently with npm v7 there's a new package-lock.json format (backward compatible with v6 but still updates the lock file every time).

Reply:
Damn that npm v7 change now makes this a net ++ PR instead of --.